capstone/src/test/javascript/calendarSpec.js

```diff
@@ -0,0 +1,20 @@
+describe('DataTable Construction', function() {
+  it('Will add correct columns/rows to DataTable from JSON', function() {
+    const mockCalenderInfo = {};
```

Comment:
Should these be 'let' instead of 'const'?

Reply:
Yea probably.